hnn_core/io.py

```diff
@@ -0,0 +1,394 @@
+"""Network io"""
+
+# Authors: Rajat Partani <rajatpartani@gmail.com>
+#          Mainak Jas <mjas@mgh.harvard.edu>
+#          Nick Tolley <nicholas_tolley@brown.edu>
+
+import os
+from h5io import write_hdf5, read_hdf5
+
+from .cell import Cell, Section
+from .cell_response import CellResponse
+from .externals.mne import fill_doc
+
+
+def str_to_node(node_string):
+    node_tuple = node_string.split(',')
+    node_tuple[1] = int(node_tuple[1])
+    node = (node_tuple[0], node_tuple[1])
+    return node
+
+
+@fill_doc
+def write_network(net, fname, overwrite=True, save_unsimulated=False):
+    """Write network to a file.
+
+    Parameters
+    ----------
+    %(net)s
+    %(fname)s
+    %(overwrite)s
+    %(save_unsimulated)s
+
+    Outputs
+    -------
+    A hdf5 file containing the Network object
+
+    File Content Description
+    ------------------------
+    %(network_file_content_description)s
+
+    cell type description
+    ---------------------
+    %(cell_description)s
+
+    gid range description
+    ---------------------
+    %(gid_range_description)s
+
+    external drive description
+    --------------------------
+    %(external_drive_description)s
+
+    external bias description
+    -------------------------
+    %(external_bias_description)s
+
+    connection description
```

Comment:
I don't see any of this getting rendered here: https://output.circle-artifacts.com/output/job/0f0c9fc8-ab90-49f1-90eb-558f1ecf063b/artifacts/0/html/generated/hnn_core.Network.html#hnn_core.Network.write

Reply:
you might want to try having a separate element for each argument ... it will also maximize reusability

Reply:
I made this change. But no matter how much I try it is not rendering headings correctly. For example - I want `cell type description` as a heading same as `Parameters` but it is not rendering. I read some numpy docs that only special keywords such as `Parameters`, `Returns`, `Yields` etc are rendered as headings - https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard.